### PR TITLE
Add the two-level header to Manage (v2)

### PR DESCRIPTION
**Pull request by duncanjbrown**
## Context

This PR implements the lower-level grey nav bar in the prototype.

## Changes proposed in this pull request

- Add `moj-frontend` bits
- Parameterize the `PhaseBanner` component so it can have its border removed when stacked on top of another element
- Refactor/add `NavigationItems#for_provider_account_nav` and `NavigationItems#for_primary_account_nav`
- touch it in a system spec

## Link to Trello card

https://trello.com/c/OTojnjhu/2440-implement-the-new-two-level-header-for-manage

## Things to check

- [X] This code doesn't rely on migrations in the same Pull Request
- [X] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [X] API release notes have been updated if necessary
- [X] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/components/phase_banner.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="govuk-width-container">
-  <div class="govuk-phase-banner app-!-print-display-none">
+  <div class="govuk-phase-banner <%= @stacked ? 'app-phase-banner__stacked' : '' %> app-!-print-display-none">
```
Comment (adamsilver) [inline app/components/phase_banner.html.erb:2]: What does ‘stacked’ mean? Is it just that there's no border?

Comment (duncanjbrown) [reply app/components/phase_banner.html.erb:2]: That it's to be stacked on top of another element. Suggestion for a better name for this param v welcome!

**Comment by duncanjbrown** [comment]
> Looks great. I would just question the ‘stacked’ class name and param name. If it's literally about removing the border then I think we can be on the nose about it. ‘app-phase-banner--noborder`.
> 
> If we want to be more semantic and generic then `provider-phase-banner`.

Great — thanks. I've made it `app-phase-banner--no-border`.

**Comment by duncanjbrown** [comment]
Just noticed the underline is required when we're on a non-primary nav page. Going to fix

**Pull request merged**